tiledb/sm/array_schema/domain.cc

```diff
@@ -61,6 +61,29 @@ Domain::Domain() {
   cell_num_per_tile_ = 0;
 }
 
+Domain::Domain(
+    Layout cell_order,
+    const std::vector<shared_ptr<Dimension>> dimensions,
+    Layout tile_order)
+    : cell_order_(cell_order)
+    , dimensions_(dimensions)
+    , dim_num_((unsigned int)dimensions.size())
+    , tile_order_(tile_order) {
+  // Compute number of cells per tile
+  compute_cell_num_per_tile();
+
+  // Compute number of cells per tile
+  set_tile_cell_order_cmp_funcs();
+
+  // Set tile_extent to empty if cell order is HILBERT
```

Comment:
Shouldn't the `tile_extent` be default-initialized to empty? Trying to understand if/why we still need this.

Reply:
Yes, we don't need this. just removed them